lighthouse-core/audits/byte-efficiency/total-byte-weight.js

```diff
@@ -19,8 +19,8 @@ const UIStrings = {
   'Large network payloads cost users real money and are highly correlated with ' +
   'long load times. [Learn ' +
   'more](https://web.dev/total-byte-weight).',
-  /** Used to summarize the total byte size of the page and all its network requests. The `{totalBytes}` placeholder will be replaced with the total byte sizes, shown in kilobytes (e.g. 142 KB) */
-  displayValue: 'Total size was {totalBytes, number, bytes}\xa0KB',
+  /** Used to summarize the total byte size of the page and all its network requests. The `{totalBytes}` placeholder will be replaced with the total byte sizes, shown in kibibytes (e.g. 142 KiB) */
+  displayValue: 'Total size was {totalBytes, number, bytes}\xa0KiB',
```

Comment:
I was going to say we should really transition these to `Intl.NumberFormat` with its new support for localization of numbers+unit, but [it only supports `kilobyte`, not `kibibyte`](https://tc39.es/ecma402/#sec-issanctionedsimpleunitidentifier) :(